### DISCUSSION TITLE
Access to command-line arguments in Numbat

### DISCUSSION THREAD

**Discussion by justind000** [sharkdp/numbat#258] [Ideas]
I think adding a way for a script to get a command-line argument would be useful. 

The use-case might be something like this: a script has a function that converts `x` to `y`. You would run the script `numbat script.nbt x`. The script would then convert `x` to `y`. 

Or allowing something like [cin](https://www.geeksforgeeks.org/cin-in-c/) to take prompted input.

**Comment by sharkdp** [comment]
I agree. Basic I/O would certainly be helpful. See also https://github.com/sharkdp/numbat/issues/262

**Comment by ginabythebay** [comment]
Is it possible to load a script from the command line and also specify something to execute? I’m thinking I would define units and functions in the script and then also specify a call to the function from the command line invocation

**Reply by sharkdp** [reply]
That is currently not possible. Please feel free to open a new ticket. This discussion is about something else.
